.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846 # tag=v3.0.0
+
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@3f943b86c9a289f4e632c632695e2e0898d9d67d # tag=v1
```

Comment:
don't we have a more explicit version? 😕

Reply:
Unfortunately not. The only tag offered right now is v1[^1]. I've opened an issue upstream[^2]

[^1]: https://github.com/actions/dependency-review-action/tags
[^2]: https://github.com/actions/dependency-review-action/issues/21